databricks/koalas/indexes.py

```diff
@@ -2783,6 +2805,31 @@ def get_level_values(self, level):
         )
         return ks.DataFrame(internal).index
 
+    def item(self):
+        """
+        Return the first element of the underlying data as a python tuple.
+
+        Returns
+        -------
+        tuple
+            The first element of MultiIndex.
+
+        Raises
+        ------
+        ValueError
+            If the data is not length-1.
+
+        Examples
+        --------
+        >>> kmidx = ks.MultiIndex.from_tuples([('a', 'x')])
+        >>> kmidx.item()
+        ('a', 'x')
+        """
+        result = self._internal.spark_frame.head(2)
```

Comment:
I think we'll have to use `to_pandas`. For example, I think it will return a different type of instance when you use timestamp. `datetime.datetime` from PySpark vs `datetime64[ns]` from pandas IIRC.

Reply:
Oh yeah, I missed it. Thanks for the catch out!